contrib/examples/cluster-operator-template.yaml

```diff
@@ -338,7 +245,7 @@ objects:
   kind: APIService
   metadata:
     name: v1alpha1.clusteroperator.openshift.io
-    namespace: ${KUBE_SYSTEM_NAMESPACE}
+    namespace: kube-system
```

Comment:
needs to be parameterized

Reply:
The parameter was commented with "don't change this", it seems like we should not treat as a parameter at that point. Does this actually need to change?

Reply:
No, it doesn't need to change, but it needs to be a parameter because of the way `oc process` works. Namespace attributes that are not parameterized are stripped when processing the template.

Reply:
Aha I understand, thanks I will update.